src/Components/CardHorizontal/CardHorizontal.jsx

```diff
@@ -18,6 +18,16 @@ const CardHorizontal = ({ product }) => {
     dispatch({ type: "REMOVE_FROM_CART", payload: _id });
   };
 
+  const isPresentInWishlist = wishlist.find(
+    (wishlistItem) => wishlistItem._id === product._id
+  );
+
```

Comment:
this function is already created in Card.js. Its better to move all these helper functions in one seperate file and then import from there

Reply:
done here:
https://github.com/saadahmadhere/elmart/pull/6/commits/7f0318c693558409d02bd8ff8ccf0343eb916664